shared/src/business/entities/cases/Case.test.js

```diff
@@ -124,6 +220,16 @@ describe('Case entity', () => {
       expect(Object.keys(myCase)).not.toContain('associatedJudge');
     });
 
+    it('does not create a secondary contact when one is not needed by the party type', () => {
```

Comment:
I'm not clear on why this is located in a describe block for "filtered" - is that intentional?

Reply:
Good catch, that was an accident!

Reply:
Updated!